chart/compass/charts/pairing-adapter/templates/virtual-service.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: {{ template "fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    release: {{ .Release.Name }}
```

Comment:
Should we add additional labels like `helm` or k8s labels here or something? https://github.com/kyma-incubator/compass/blob/ac249a4671d2099d7c172ce130e749bc281b7a6f/chart/compass/charts/connector/templates/virtual-service.yaml#L8